src/main/java/mate/academy/util/ConnectionUtil.java

```diff
@@ -0,0 +1,28 @@
+package mate.academy.util;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class ConnectionUtil {
+    private static final String URL = "jdbc:mysql://localhost:3306/library_service";
```

Comment:
The URL for the JDBC connection is incomplete. It should specify the database you're connecting to, for example, 'jdbc:mysql://localhost:3306/myDatabase'.